pyeudiw/storage/mongo_storage.py

```diff
@@ -225,34 +224,58 @@ def _add_entry(
         db_collection = getattr(self, collection)
         db_collection.insert_one(attestation)
         return entity_id
+    
+    def _update_attestation_metadata(self, ta: dict, attestation: list[str], exp: datetime, trust_type: TrustType):
+        trust_name = "x509" if trust_type == TrustType.X509 else "federation"
```

Comment:
that's because you use Enum, I suggest you to add a mapping like

````
trust_type_map :dict = {
  TrustType.X509 : "x509",
  TrustType.Federation: "federation"
}
````

then

````
trust_name = trust_type_map[trust_type]
````

and we remove the `if` statement